src/routes/applications.ts

```diff
@@ -287,6 +286,18 @@ const createApplicationsRouter = (
     }),
   );
 
+  router.get(
+    '/export/application-history/',
+    authFilter([config.auth.REVIEW_SCOPE]),
+    wrapAsync(async (req: Request, res: Response) => {
+      const apps = await getApplicationUpdates();
+      const tsv = createAppHistoryTSV(apps);
```

Comment:
can't these be one call ? It's always a good practice to keep the router as thin as possible and keep everything pushed in the service

Reply:
will do!

Reply:
i even did that for the csv call, not sure why i did it differently here 🤔